examples/tests/cypress/integration/channels.spec.js

```diff
@@ -16,11 +16,11 @@ describe('Channels', () => {
       cy.subscribe('mike', 'message');
       // "name":"mike","foo":"bar" due to way how Cypress handles escaping of curly braces
       cy.sendEvent('message', '"name":"mike","foo":"bar"');
-      cy.assertReceivedEvents('event-log', 'message', '"name":"mike","foo":"bar"');
+      cy.assertReceivedEventLog('event-log', '"name":"mike","foo":"bar"');
       // send constrained event for different user - John
       cy.sendEvent('message', '"name":"john","foo":"bar"');
       // Mike shouldn't receive any new event
-      cy.wait(2000).assertReceivedEvents(
```

Comment:
2s wait was there to make sure that no events were received even after some time. If you don't wait at least a bit you can't be sure that no events arrived. 2s give us at least some confidence.

Same goes for other occurrences.

Reply:
I mean...the timeouts basically achieve the same thing, right? As I see it, `wait` just adds to the tests flakyness.

Reply:
No they don't, timeouts just say how long to wait till e.g. element is visible (e.g. it's not empty but contains one `li` element or whatever). Here we are testing that even after some time there is still one event (so you already start with one `li` and expect it won't be 2 or 3 therefore the wait period). So without waiting, you are actually testing nothing.

Reply:
The problem is that even 2 seconds don't guarantee anything. If the browser is busy or the CI is just slow, even after 2 seconds of wait time the test isn't sound.

Reply:
I can put the `wait` back and increase the wait time but we should really be looking for a long term solution.

Reply:
I agree (mentioned it in the first comment), but it's not that easy, especially when cypress doesn't support by default multi-tabs, multi-windows and you have to put it together with Selenium or something.

Reply:
Should I just re-add the `wait`?

Reply:
@mmacai I have re-added `wait` for now. The tests are not flaky anymore, which was the priority. I will work on actually fixing the test (with the above mentioned method; two connections) at a later point.

Reply:
@mmacai Okay...it got flaky again. Apparently the `wait` does kinda benefit to the test being flaky ☹️

Any ideas on that?

Reply:
Hmm, that's strange, don't see any reason why it would cause flakiness, since it's pretty much just sitting there and doing nothing. Did you save the error log from Travis by any chance?